documentation/index.md

```diff
@@ -1,48 +1,33 @@
-- Basix ([latest docs](https://docs.fenicsproject.org/basix/main/)) is a element definition and tabulation library
-  that provides all the information FEniCSx needs about elements on the reference cell.
-- FFCx ([latest docs](https://docs.fenicsproject.org/ffcx/main)) (the FEniCSx Form Compiler) is the Python library that interprets
-  UFL forms and generates C code to assemble these on cells.
-- DOLFINx ([latest C++ docs](https://docs.fenicsproject.org/dolfinx/main/cpp/), [latest Python docs](https://docs.fenicsproject.org/dolfinx/main/python/))
-  is the main user interface of FEniCSx, and handles meshes and linear algebra solvers among other things.
-
-API documentation of the latest version of each component can be found using the links above. Documentation of
-other versions can be found at [docs.fenicsproject.org](https://docs.fenicsproject.org/).
 
 # The FEniCS book
 ![The FEniCS book](/assets/img/docs/book.png){: .image-right }
-The book [Automated Solution of Differential Equations by the Finite Element Method](book.md)
-explains the theoretical background and design of FEniCS. It describes the FEniCS software
-components in detail and showcases a number of applications of FEniCS to problems in fluid
-mechanics, solid mechanics, electromagnetics, and geophysics. The book was published in 2012,
-and was based on the legacy FEniCS library, so the code examples in the book are out of date.
-However, the book still gives a good description of the many of the design principles that
+The book [Automated Solution of Differential Equations by the Finite
+Element Method](book.md) explains the theoretical background and design
+of FEniCS. It describes the FEniCS software components in detail and
+showcases a number of applications of FEniCS to problems in fluid
+mechanics, solid mechanics, electromagnetics, and geophysics. The book
+was published in 2012, and was based on the legacy FEniCS library, so
+the code examples in the book are out of date. However, the book still
+gives a good description of the many of the design principles that
 FEniCSx is based on.
 
-The book is available as a [free ebook](http://launchpad.net/fenics-book/trunk/final/+download/fenics-book-2011-10-27-final.pdf),
-or can be bought from [Springer](http://www.springer.com/mathematics/computational+science+%26+engineering/book/978-3-642-23098-1)
+The book is available as a [free
+ebook](http://launchpad.net/fenics-book/trunk/final/+download/fenics-book-2011-10-27-final.pdf),
+or can be bought from
+[Springer](http://www.springer.com/mathematics/computational+science+%26+engineering/book/978-3-642-23098-1)
 or many other bookshops.
 
-## Legacy FEniCS
-Documnetation for the legacy version of FEniCS (version {{ site.fenicsversion }}) can be found [here](archive.md).
+
+## Legacy FEniCS documentation
+
+Documnetation for the legacy version of FEniCS (version {{
+site.fenicsversion }}) can be found [here](archive.md).
```

Comment:
Links to archive.md here. Did you mean to delete archive.md?

Reply:
Yes and no I didn't see it was still linked to. `archive.md` has a lot of text - I'll see if it can be replaced by a few links.